packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -142,6 +143,16 @@ function extractManifestOutput(output: string): string {
   return stripAnsi(output.substring(start, end + 1));
 }
 
+export function populateMetadata(angularVersion?: string): Metadata {
```

Comment:
optional nit: populateMetadata sounds like it should take another metadata object as a param and assign values to fields. Can we rename this createMetadata or produceMetadata?